streaming/src/main/scala/org/apache/gearpump/streaming/task/Subscription.scala

```diff
@@ -109,6 +109,11 @@ class Subscription(
         transport.transport(ackRequest, targetTask)
       }
 
+      if (messageCount(partition) % maxPendingMessageCount == 0) {
```

Comment:
I think the story is different compared with ackRequest frequency setting.

For ackRequest setting, the size will impact the memory footprint in Mailbox. 
For ProbeLatency, it is not necessary to expose this concept to user, I think.